code/__DEFINES/flags.dm

```diff
@@ -72,6 +72,7 @@
 #define EASYDISMEMBER	20
 #define EASYLIMBATTACHMENT 21
 #define TOXINLOVER		22
+#define DIGITIGRADE		23	//Uses weird leg sprites. Optional for Lizards, required for ashwalkers. Don't give it to other races unless you make sprites for this (see human_parts_greyscale.dmi)

```

Comment:
For the record I am not a furry and I'm pretty sure WJ isn't either 

unless xenomorphs count, in which case uncertainty exists within my mind